packages/provider/src/RelayClient.ts

```diff
@@ -95,6 +95,7 @@ export class RelayClient {
     this.rawConstructorInput = rawConstructorInput
     this.logger = rawConstructorInput.overrideDependencies?.logger ??
       createClientLogger(rawConstructorInput.config?.loggerConfiguration ?? defaultLoggerConfiguration)
+    this.logger.info(`Created new RelayClient ver.${gsnRuntimeVersion}`)
```

Comment:
any special reason to report it twice ? we always have both RelayClient and RelayProvider (and RelayClient is basically an internal class...)